Cargo.toml

```diff
@@ -100,7 +100,12 @@ normal = ["workspace-hack"]
 development = ["expect-test", "pretty_assertions"]
 
 [workspace.dependencies]
-foyer = { version = "0.16.1", features = ["serde", "tracing", "nightly"] }
+# foyer = { version = "0.16.1", features = ["serde", "tracing", "nightly"] }
+foyer = { git = "https://github.com/foyer-rs/foyer", rev = "fb8030ead6570d62f115e9a7fc442bc874aadcea", features = [
+    "serde",
+    "tracing",
+    "nightly",
+] }
```

Comment:
why not publish

Reply:
My usual workflow is to first conduct longevity testing in RisingWave, resolve some obvious bugs, and then release the foyer version, followed by bumping the foyer version on the RW side. This is just a temporary placeholder. I will modify the release version before merging. Let me leave a comment here. 🙏